nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -39,10 +31,12 @@ in
         type = types.listOf types.package;
       };
 
-      token = mkOption {
-        type = types.str;
+      tokenPath = mkOption {
+        type = types.path;
         description = ''
           The token from your Buildkite "Agents" page.
+
+          A path to the token file.
```

Comment:
Needs clearer explanation that the path is used at runtime, not build time.

Reply:
Done.